testsuite/__init__.py

```diff
@@ -74,5 +74,6 @@ def _settimeout(self, timeout):
     os.environ["OPENSHIFT_CLIENT_PYTHON_DEFAULT_SKIP_TLS_VERIFY"] = "true"
 
 TESTED_VERSION = Version(str(settings["threescale"]["version"]))
+TESTED_APICAST_VERSION = Version(str(settings["threescale"]["gateway"]["OperatorApicast"]["version"]))
```

Comment:
this isn't good place. I suggest `["threescale"]["apicast_operator_version"]`

Reply:
Moved to suggested path